doc/sphinx/install.rst

```diff
@@ -8,13 +8,81 @@ Installing Cantera
    :local:
    :depth: 2
 
+.. _sec-install-conda:
+
+Conda
+=====
+
+`Anaconda <https://www.continuum.io/downloads>`_ and `Miniconda
+<http://conda.pydata.org/miniconda.html>`_ are Python distributions for which
+Cantera is available through the `conda` package manager. Both distributions are
+available for Linux, OS X, and Windows. The base Anaconda distribution includes
+a large number of Python packages that are widely used in scientific
+applications. Miniconda is a minimal distribution, where all of the packages
+available in Anaconda (and more) can be installed using the package manager.
+Note that installing Cantera using conda will only provide the Cantera Python
+module. If you want to use the other Cantera interfaces, see the OS-specific
+installation options below.
+
+For more details on how to use conda, see the `conda documentation
+<http://conda.pydata.org/docs/intro.html>`_.
+
+**Option 1: Create a new environment for Cantera**
+
+If you have just installed Anaconda or Miniconda, the following instructions
+will create a conda environment where you can use Cantera. For this example, the
+environment is named *spam*. From the command line, run::
+
+    conda create -n spam -c cantera ipython matplotlib
+
+This will create an environment with Cantera, IPython, Matplotlib, and all their
+dependencies installed. If you are running Linux or OS X, you can then activate
+this environment by running::
+
+    source activate spam
+
+If you are running Windows, the equivalent command is::
+
+    activate spam
+
+**Option 2: Install Cantera in an existing environment**
+
+First, activate your environment (assumed to be named *spam*). For Linux and OS
+X, this is done by running::
+
+    source activate spam
+
+For Windows users, the command is::
+
+    activate spam
+
+Then, install Cantera by running::
+
+    conda install -c cantera cantera
+
+**Option 3: Install the development version of Cantera**
+
+To install a recent development snapshot (i.e. an alpha or beta version) of
+Cantera in an existing environment, run::
+
+    conda install -c cantera/label/dev cantera
+
+If you later want to revert back to the stable version, first remove and then
+reinstall Cantera::
+
+    conda remove cantera
+    conda install -c cantera cantera
+
 .. _sec-install-win:
 
 Windows
 =======
 
 Windows installers are provided for stable versions of Cantera. These
-installation instructions are for Cantera 2.2.0.
+installation instructions are for Cantera 2.2.0. Use these installers if you
```

Comment:
Then maybe just `2.2` would be better. It might be confusing if we don't distribute 2.2.0 anymore. Also, if its on `master` (and presumably going into the dev-docs), should it be 2.3?